governance/program/src/state/proposal.rs

```diff
@@ -1834,6 +1832,57 @@ mod test {
         assert_eq!(proposal.max_vote_weight, Some(100));
     }
 
+    #[test]
+    fn test_try_tip_vote_with_reduced_absolute_community_mint_max_vote_weight() {
+        // Arrange
+        let mut proposal = create_test_proposal();
+
+        proposal.options[0].vote_weight = 60;
```

Comment:
As another test, can you try vote with more than the absolute voter weight?  For example, just like this one, but instead have `vote_weight = 120`.  Just to make sure that overflows are correctly handled

Reply:
Done